x-pack/plugins/security/public/views/management/roles.html

```diff
@@ -1,228 +0,0 @@
-                    ng-checked="allSelected()"
-                    ng-click="toggleAll()"
-                    aria-label="{{allSelected() ? 'Deselect all rows' : 'Select all rows'}}"
-                  >
-                </div>
-              </th>
-
-              <th
-                class="kuiTableHeaderCell"
-                scope="col"
-              >
-                <button
-                  class="kuiTableHeaderCellButton"
-                  ng-class="{'kuiTableHeaderCellButton-isSorted': sort.orderBy == 'name'}"
-                  ng-click="toggleSort(sort, 'name')"
-                  aria-label="{{sort.reverse ? 'Sort role ascending' : 'Sort role descending'}}"
-                >
-                  <span class="kuiTableHeaderCell__liner">
-                    <span
-                      i18n-id="xpack.security.management.roles.roleTitle"
-                      i18n-default-message="Role"
-                    ></span>
-                    <span
-                      aria-hidden="true"
-                      class="kuiTableSortIcon kuiIcon"
-                      ng-class="getSortArrowClass('name')"
-                    ></span>
-                  </span>
-                </button>
-              </th>
-
-              <th scope="col" class="kuiTableHeaderCell">
-                <span
-                  class="kuiTableHeaderCell__liner"
-                  i18n-id="xpack.security.management.roles.reversedTitle"
-                  i18n-default-message="Reserved"
-                ></span>
-                <span class="kuiIcon fa-question-circle" tooltip={{reversedTooltip}} aria-label={{reversedAriaLabel}} ></span>
-              </th>
-            </tr>
-          </thead>
-
-          <tbody>
-            <tr
-              ng-repeat="role in roles | orderBy:'name' | filter:query | orderBy:sort.orderBy:sort.reverse"
-              data-test-subj="roleRow"
-              class="kuiTableRow"
-            >
-              <td class="kuiTableRowCell kuiTableRowCell--checkBox">
-                <div class="kuiTableRowCell__liner">
-                  <input
-                    type="checkbox"
-                    class="kuiCheckBox"
-                    ng-click="toggle(selectedRoles, role)"
-                    ng-checked="includes(selectedRoles, role)"
-                    ng-disabled="role.metadata._reserved"
-                  >
-                </div>
-              </td>
-
-              <td class="kuiTableRowCell">
-                <div class="kuiTableRowCell__liner">
-                  <a
-                    class="kuiLink"
-                    ng-href="{{getEditRoleHref(role.name)}}"
-                    data-test-subj="roleRowName"
-                  >
-                    {{ role.name }}
-                    <span
-                      ng-if="!isRoleEnabled(role)"
-                      class="euiBadge euiBadge--default"
-                      tooltip="This role is currently disabled. You may only view or delete it."
```

Comment:
Can roles no longer be "disabled" as is called out here? Just checking for parity since I didn't see the option.

Reply:
Roles can be disabled via the Elasticsearch APIs. We've never exposed that toggle in the Kibana UI, but we do show if the role has been disabled.

Reply:
Ok, I didn't see this in the new code. It should probably be added back in, yeah?

Reply:
We have it included here: https://github.com/legrego/kibana/blob/bae75ac129a06b16ada836bd3ae8fc56acf71f3d/x-pack/plugins/security/public/views/management/roles_grid/components/roles_grid_page.tsx#L180-L192

Reply:
Can a role be both disabled and reserved?

Reply:
Not sure, I'm looking into this now

Reply:
@cchaos I finally figured it out -- it's not possible for a role to be both disabled and reserved. Disabled roles are rare: they're the result of a role containing features that aren't compatible with the current license. For example, DLS/FLS settings with a Gold license.

Reply:
Ok, so then I think the warning icon is too harsh and it should just be text like:

<img width="408" alt="Screen Shot 2019-04-08 at 14 16 13 PM" src="https://user-images.githubusercontent.com/549577/55746958-1846a800-5a09-11e9-8997-78848a2e90d7.png">

```jsx
<EuiText color="subdued" size="s">
  <EuiLink data-test-subj="roleRowName" href={getRoleManagementHref(name)}>
    {name}
  </EuiLink>
  {isRoleEnabled(record) && ' (disabled)'}
</EuiText>
```

And then on the Edit page, add that text in a callout above the name panel:

<img width="584" alt="Screen Shot 2019-04-08 at 14 21 07 PM" src="https://user-images.githubusercontent.com/549577/55747147-9c992b00-5a09-11e9-8ef2-60fdb047370f.png">